### ISSUE TITLE
The performance of NNMF

### ISSUE THREAD

**Issue by zhibin-daily** [jstol/neural-net-matrix-factorization#1]
Hi, Stolee,
      I use your code to reproduce the NNMF. However, I can't get the same experimental results as your paper "Matrix Factorization with Neural Networks and Stochastic Variational Inference".  The test RMSE 
(ML-1M) of NNMF is 0.8539, my experimental result is 0.8725 with the parameters (lambda = 1 (Obtained by hyper-parameter selection), learning rate = 0.005, batch-size = 25000). Can you tell me the parameters(RMSE 0.8539  on the ML-1M)?
